package.json

```diff
@@ -5,9 +5,14 @@
   "main": "index.js",
   "type": "module",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "lint": "npx prettier . --check",
+    "lint:fix": "npx prettier . --write"
   },
   "keywords": [],
-  "author": "",
-  "license": "ISC"
+  "author": "Alan Castillo Esquina",
+  "license": "ISC",
+  "dependencies": {
+    "prettier": "^3.1.0"
```

Comment:
There's a type error, remove the "^"

Reply:
It's not a typo error, it means you can install a compatible version of 3.1.0

Reply:
Keep working 🚀